app/services/catalog/determine_task_relevancy.rb

```diff
@@ -31,8 +31,6 @@ def delegate_task
       elsif @task.context.has_key_path?(:applied_inventories)
         Rails.logger.info("Creating approval request for task")
         Catalog::CreateApprovalRequest.new(@task).process
-      else
-        add_task_update_message
```

Comment:
Do we want to allow this to potentially not log or do anything? I liked having the guard `else` here. At the very least so we have some recompense for information.

Reply:
Well, we're always logging on line 16 now, so we still have the information.

Reply:
Would there be value in adding:
```
Rails.logger.info "... delegate task failed "
```
On line 16 we'll see that `delegate_task` was called but the steps following delegate task never completed.
Logging if for any reason we fall out of the conditional seems at worst: helpful.

Reply:
Yeah that's true, I'll add a line.